mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -672,6 +672,17 @@ LogicalResult PopulateParamsXDL::isValidBlockwiseGemm(
     return failure();
   }
 
+  // Reject invalid blockSize
+  int64_t kPerBlock = param.gemmKPerBlock;
+  int64_t mPerBlock = param.gemmMPerBlock;
+  int64_t nPerBlock = param.gemmNPerBlock;
+  int64_t aCopyPerThread = (kPerBlock * mPerBlock) / blockSize;
+  int64_t bCopyPerThread = (kPerBlock * nPerBlock) / blockSize;
+  if (aCopyPerThread == 0 || bCopyPerThread == 0) {
+    LLVM_DEBUG(llvm::dbgs() << "tuning: Block size too large.\n");
+    return failure();
+  }
```

Comment:
This needs to be in sync : https://github.com/ROCm/rocMLIR/blob/225f6b78ab260de8523a92f1c7a8b31426d2bc9d/mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp#L433C3-L437C6

Therefore, how about refactor this into a function in a common util ?